relays/substrate/src/rialto_millau/cli.rs

```diff
@@ -47,6 +47,17 @@ pub enum RelayHeaders {
 		#[structopt(flatten)]
 		prometheus_params: PrometheusParams,
 	},
+	/// Relay Westend headers to Millau.
+	WestendToMillau {
```

Comment:
I was imagining this will be part of a separate `westend_millau` module, but I can refactor this later on.